docs/provider/conjur.md

```diff
@@ -21,11 +21,18 @@ If you set up your Conjur server with a self-signed certificate, we recommend th
 {% include 'conjur-ca-bundle.yaml' %}
 ```
 
-### External secret store with apiKey authentication
+### External secret store
```

Comment:
@szh - not sure if we need this section, does it anything that isn't stated above?